pi_piper-sysfs.gemspec

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pi_piper/sysfs/version'
```

Comment:
Alright sure, I will create a gist to hopefully explain it better. I think some code samples will make this clearer.